src/new-client/testing/log-in.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect } from '@playwright/test';
+import env from './testEnvironment';
+
+const { TEST_USER_USERNAME, TEST_USER_PASSWORD } = env;
+
+test('Loads the site', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page).toHaveTitle('Appointment Management Service');
+});
+
+test('Has a log in link', async ({ page }) => {
```

Comment:
Could/should be covered by a component test? Get faster feedback on this by testing lower down.

Reply:
Will this get implicitly tested by the login/logout tests in the sense that they will fail if that link is not visible? Or even add the assertion to that test to make it more explicit if needed? I'm just wondering on whether a separate test is needed for this?

Reply:
True enough - how about if I remove this test altogether, but add an extra assertion to the "logs in" test to prove that the "Log out" button is visible (& also maybe that "Log in" isn't)?

Reply:
As discussed on our call, have removed but have added in assertions to the other tests